localstack/services/infra.py

```diff
@@ -269,7 +269,8 @@ def start_proxy(port, backend_url, update_listener=None, quiet=False, params={},
 def start_moto_server(key, port, name=None, backend_port=None, asynchronous=False, update_listener=None):
     if not name:
         name = key
-    print('Starting mock %s (%s port %s)...' % (name, get_service_protocol(), port))
+    print('Starting mock %s service in %s ports %s (recommended) and %s (deprecated)...' % (
+        name, get_service_protocol(), DEFAULT_PORT_EDGE, port))
```

Comment:
Can we please replace all occurrences of `DEFAULT_PORT_EDGE` in this pull request with `config.EDGE_PORT`. Note that `config.EDGE_PORT` defaults to `DEFAULT_PORT_EDGE`, but can also be configured/customized by the user.

Reply:
Sure!

Reply:
I have the changes locally already, but I asked you a little doubt about them in gitter

Reply:
Thanks @arcones , I saw your message, but `config.EDGE_PORT` is the source of truth. If you're seeing a different port, that may be related to your local configuration, or perhaps if you're using the Pro version with an API key? In any case, `config.EDGE_PORT` should be the definitive source of truth of the actual edge API port.